test/solve_pnp.cpp

```
@@ -1,424 +1,322 @@
+    auto distort_coeff = std::array<double, 5> { k1, k2, 0, 0, k3 };
+    PnpSolution::Input input;
+    input.camera_matrix = { {
+        { fx, 0.0, cx },
+        { 0.0, fy, cy },
+        { 0.0, 0.0, 1.0 },
+    } };
+    input.distort_coeff = distort_coeff;
 
-    EXPECT_TRUE(is_quaternion_normalized(solution.result.orientation)) //
-        << "Quaternion should be normalized";
+    return input;
```

Cppcheck (2.18.0)

[error] Uninitialized variables

(uninitvar)